### DISCUSSION TITLE
Why use `markdown-it` by default when `marked` looks better

### DISCUSSION THREAD

**Discussion by Muhammed-Rahif** [11ty/eleventy#3556] [Uncategorized]
Firstly and forever, no hate towards to `markdown-it`, I absolutely love any open source.
But I wanted to know what is the motive behind using `markdown-it`, when `marked` have:

| | `marked` | `markdown-it` |
| --- | ----------- | ------------------- |
| npm downloads | 12,987,644+ | 7,544,135+ |
| Github stars | 33.2k+ | 18.3k+ |
| Last updated | 3 days ago | 8 months ago |
| Dependencies | 0 Dependencies | 6 Dependencies |
| Dev Dependencies | 25 Dev Dependencies | 31 Dev Dependencies |
| Unpacked Size | 953 kB | 767 kB |

I see both `marked` and `markdown-it` in https://github.com/11ty/eleventy/blob/main/package.json. `marked` in "devDependencies" and `markdown-it` in "dependencies", even though `markdown-it` is used by default.

_Is there anything that I am missing?_

**Comment by nhoizey** [comment]
I don't know the answer to this question.

But I've been using Markdown-it and multiple of its plugins in my Eleventy projects for many years now, like many other people, so I'm not sure changing the default would be easy.

**Comment by zachleat** [comment]
The tests include an example of swapping the markdown engine from `markdown-it` to `marked` (which is why `marked` is a dev dep)

**Comment by bobmonsour** [comment]
I do know that some, including me, in the Eleventy community use a variety of markdown-it plugins, and I expect that they would not map well into a marked environment. You might want to take a look at these two comparison articles. I think that the plugins for markdown-it are the key here.

https://moiva.io/?npm=markdown-it+marked

https://npm-compare.com/markdown-it,marked,remark,showdown
